app/javascript/packs/footer.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faYoutube, faTwitter } from '@fortawesome/free-brands-svg-icons';
+import Link from '../components/link';
+
+const Footer = () => (
+  <div className="footer">
+    <div className="container fgs-al  fgs-al-align-items-flex-start  fgs-al-justify-content-space-between">
+      <div className="company-info caption-light fgs-ali fgs-al fgs-al-v fgs-al-g-10 ">
+        <span className="title caption-bold fgs-ali">BiteSite Inc.</span>
+
+        <div className="info fgs-al fgs-ali  fgs-al-v fgs-al-g-6">
+          <span className="info fgs-ali">© 2021 BiteSite Inc.</span>
+
+          <span className="info fgs-ali">
+            204 - 78 George St.
+            <br />
+            Ottawa, ON <br /> K1N 5W1
+          </span>
+          <span className="info fgs-ali">info@bitesite.ca</span>
+        </div>
+      </div>
+
+      <div className="links fgs-al fgs-ali  fgs-al-h fgs-al-align-items-flex-start fgs-al-g-60 ">
+        <div className="company-links caption-medium fgs-ali fgs-al fgs-al-v fgs-al-g-6">
+          <span className="links-title fgs-ali">COMPANY</span>
+          <Link path="/services" className="fgs-ali">
+            Services
```

Comment:
@NRaeewal what do you think if all links would open a new tab? can we add this option?